Library/Homebrew/livecheck/strategy/crate.rb

```diff
@@ -97,7 +97,13 @@ def self.find_versions(url:, regex: nil, provided_content: nil, homebrew_curl: f
           content = if provided_content
             provided_content
           else
-            match_data.merge!(Strategy.page_content(match_data[:url], homebrew_curl:))
+            match_data.merge!(
+              Strategy.page_content(
+                match_data[:url],
+                url_options:   unused.fetch(:url_options, {}),
```

Comment:
- `unused` is not `unused` any longer: it's literally used here
- `unused.fetch` implies this is a `T::Hash[Symbol, T.untyped]` and should be typed accordingly

Reply:
> As mentioned in https://github.com/Homebrew/brew/pull/19077#discussion_r1913534358, the Sorbet type for rest parameters refers to the type of the hash value, so unused: T.untyped means T::Hash[Symbol, T.untyped].

Ok, thanks for clarifying, this isn't an issue then.

> The `unused` name isn't great (I optimized for brevity) but it covers two types of values:

I think this is the main thing I have a problem with that's blocking this PR being merged. Let's use a name that'd not `unused` as, at least to me, that strongly implies a parameter that will, y'know, be unused 😁

Reply:
This isn't a new issue and `unused` won't be around much longer, so it would be nice to not have to rename the 65 instances of `unused` across 23 strategies if it isn't absolutely necessary here. I've already done something similar in local branches as part of eliminating `unused` and it's not very fun 😆

Let me tinker with my `options` branch to see if I can pull out the `curl_args` bits (see https://github.com/Homebrew/brew/pull/19233#discussion_r1945041244), as that would allow me to create a PR for it right after this one. I could do the cached `content` PR right after that one and it _should_ let us get rid of `unused` as an added bonus. I'll let you know.

Reply:
if it'll be literally your next PR after this one: fine with me to merge without this change.